### PR TITLE
feat #12: added ability to manage block state

### PR DESCRIPTION
**Pull request by y9san9**

### CODE REVIEW

File: web3/src/commonMain/kotlin/dev.icerock.moko.web3/BlockState.kt [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package dev.icerock.moko.web3
+
+import kotlin.jvm.JvmInline
+
+
+interface BlockState {
```
Comment (Alex009) [inline web3/src/commonMain/kotlin/dev.icerock.moko.web3/BlockState.kt:6]: Sealed interface will be better here

File: web3/src/commonMain/kotlin/dev.icerock.moko.web3/BlockState.kt [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package dev.icerock.moko.web3
```
Comment (Alex009) [inline web3/src/commonMain/kotlin/dev.icerock.moko.web3/BlockState.kt:1]: Copyright required

**Review by Alex009 — CHANGES_REQUESTED** [review]

File: web3/src/commonMain/kotlin/dev.icerock.moko.web3/BlockState.kt (hunk 2 above)
Comment (y9san9) [reply web3/src/commonMain/kotlin/dev.icerock.moko.web3/BlockState.kt:5]: Added

File: web3/src/commonMain/kotlin/dev.icerock.moko.web3/BlockState.kt (hunk 1 above)
Comment (y9san9) [reply web3/src/commonMain/kotlin/dev.icerock.moko.web3/BlockState.kt]: Added

**Review by Alex009 — APPROVED** [review]

**Pull request merged**